compatLib/build.gradle

```diff
@@ -4,6 +4,17 @@ plugins {
 
 android {
     namespace "app.lawnchair.compatlib"
+
+    buildFeatures {
+        aidl true
+    }
+
+    sourceSets {
+        main {
+            java.srcDirs = ["src"]
+            aidl.srcDirs = ['src']
+        }
+    }
```

Comment:
Is there any specific reason to move these default source sets?

Reply:
No, does this causing an issue?

Reply:
Nope, if not, I suggest we can put them back.